jekyll/_includes/partials/notes/standalone-unsupported.adoc

```diff
@@ -0,0 +1 @@
+NOTE: This feature is not currently supported for GitLab, GitHub App or Bitbucket Data Center projects. To find out if you authorized through the GitHub OAuth app or the CircleCI GitHub App, and an overview of feature support, see the xref:version-control-system-integration-overview#feature-support-for-each-integration-type[VCS integration overview] page.
```

Comment:
I'm worried that this phrasing will make customers think that a project that uses GitHub App pipelines in a **classic** GitHub org will be excluded from the features that include this disclaimer. 
But I can't think of an easy way to easily communicate the difference between "classic" and "pipeline" without complex references to the sign-in method. 

I almost wonder if what we really need is an explicit definition of org "type" in org settings

Reply:
good point. maybe "/circleci" in the URL ? im a bit stumped, it's messy. l'm looking at this list https://circleci.com/docs/github-apps-integration/#currently-not-supported

Restrict a context to a security group: works for gh app pipelines in an oauth app org/project. doesnt work for gh app pipelines in a standalone ("/circleci") org

In-app config editor: doesn't work for hybrid nor standalone

Account integrations: works for hybrid but not for standalone

Scheduled pipelines: you cant schedule gh app pipelines in a classic org, but you can schedule oauth app pipelines. doesnt work for standalone

Reply:
So say that we had a definition of org type:
- Gen1 (= `/github/`, `/bitbucket/`)
- Gen2 (= `/circleci/`)

Then we'd be able to simplify the above like this:
- Restrict a context to security group: **only available to gen1 orgs**
- Account integrations: **only available to gen1 orgs**
- Scheduled pipeline: **only available to gen1 orgs**
- In-app config editor: **only available on GitHub OAuth and Bitbucket Cloud pipelines**

Reply:
Obviously the name gen1 gen2 would need workshopping, but if we just you "org type" in project settings that'd make things infinitely easier!

Reply:
Trigger rules is a new  feature that isn't out yet - listed it out to test how it would fit with this model